src/pages/patients/Login.js

```diff
@@ -103,6 +107,8 @@ function SignIn() {
         setIsLoading(false);
         toast.dark(t('authenticationRequired'));
       }
+    } else {
+      setIsFormInComplete(true);
```

Comment:
```suggestion
toast.warning(t('fillRequiredFields'));
```
We use toasts, maybe a new toast  should follow the same pattern in application